pandas/tests/series/test_timeseries.py

```diff
@@ -1012,3 +1012,22 @@ def test_get_level_values_box(self):
         index = MultiIndex(levels=levels, labels=labels)
 
         assert isinstance(index.get_level_values(0)[0], Timestamp)
+
+    def test_diff_sub_timedelta(self):
```

Comment:
These tests go in tests/scalar/timedelta/test_arithmetic.py

Reply:
Will move!